src/queries/match.ts

```diff
@@ -0,0 +1,23 @@
+import { useMutation, useQuery, useQueryClient } from 'react-query'
+import { handleAxiosError } from './handleAxiosError'
+import * as matchesService from '../services/match'
+
+export const useMatches = () => {
```

Comment:
Maybe the hook could receive a `onError` callback parameter? 🤔

Reply:
Do you mean like this?

```
export const useMatches = ({onError}) => {
  const matchesQuery = useQuery('matches', matchesService.getMatches, {
    staleTime: 120 * 100,
    onError: onError,
  })

  return matchesQuery
}
```
I'm not sure if I understood.